pkg/util/resizefs/resizefs_linux.go

```diff
@@ -84,7 +90,7 @@ func (resizefs *ResizeFs) Resize(devicePath string) (bool, error) {
 		}
 		return resizefs.xfsResize(devicePath)
```

Comment:
I think the resize operation maybe can't work correctly for `xfs` file system: `resizeFileSystem` method is performed between attaching device and mounting device, it means the resize operation maybe performed before the device mounted. But `xfs_growfs` requires the device has been mounted. So if we try to resize a volume with `xfs` file system, we may receive an error event like this:

```
MountVolume.resizeFileSystem failed for volume "pvc-60265674-0406-11e8-9c91-0800273c9701" : resize of device /dev/rbd0 failed: exit status 1. xfs_growfs output: xfs_growfs: /dev/rbd0 is not a mounted XFS filesystem
```

Reply:
Thank you for catching this. It is fixed now.